input/input.go

```diff
@@ -77,3 +82,9 @@ func FLBPluginGetCMetricsContext(plugin unsafe.Pointer) (*cmetrics.Context, erro
 	cmt := unsafe.Pointer(ctx)
 	return cmetrics.NewContextFromCMTPointer(cmt)
 }
+
+func FLBPluginLogPrint(plugin unsafe.Pointer, log_level C.int, message string) {
```

Comment:
I see this function does not have a `//export` comment for C. So it must be used from Go side.
What shall we pass as `plugin` here then?

Reply:
This unsafe.Pointer should be got on `FLBPluginInit` not register.